packages/react-components/react-popover/src/stories/Popover/PopoverDefault.stories.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Button } from '@fluentui/react-button';
 import { makeStyles } from '@griffel/react';
 
-import { Popover, PopoverTrigger, PopoverSurface, PopoverProps } from '../index';
+import { Popover, PopoverTrigger, PopoverSurface, PopoverProps } from '@fluentui/react-popover';
```

Comment:
_nit:_ You could split these up into

```suggestion
import { Popover, PopoverSurface, PopoverTrigger } from '@fluentui/react-popover';
import type { PopoverProps } from '@fluentui/react-popover';
```